doc/source/conf.py

```diff
@@ -54,7 +54,9 @@
 # The short X.Y version.
 version = '5.7'
 # The full version, including alpha/beta/rc tags.
-release = '5.7.26-29'
+
+release = '5.7.27-30'
+
```

Comment:
why extra empty lines?

Reply:
Probably from the conflict resolution in the merge. I will fix.

Reply:
fixed.